front-page.php

```diff
@@ -175,9 +165,10 @@
 
       <div class="col s8">
 
-        <img src="https://placeholdit.imgix.net/~text?txtsize=50&txt=AD&w=400&h=400">
-
-        <img src="https://placeholdit.imgix.net/~text?txtsize=50&txt=AD&w=400&h=400">
+        <div class="feed-post-container col s6">
```

Comment:
oh wait. then better to have an empty s2 as the spacing element. then you don't need to override anything